docs/meshstack.gcp.index.md

```diff
@@ -206,12 +206,18 @@ let GcpPlatformCoreConfiguration =
         The resulting string must not exceed a total length of 30 characters. Only alphanumeric + hyphen are allowed.
         We recommend that configuration include at least 3 characters of the random parameter to reduce the chance of
         naming collisions.
+
+      allowHierarchicalFolderAssignment:
+        Configuration flag to enable or disable hierarchical folder assignment in GCP. This means
+        projects can be assigned to sub folders of the defined resource manager folder in a landing zone. 
+        If this config flag is disabled the replicator force to create projects only directly under the resource manager folder.
```

Comment:
I thought the replicator would also move a project to the parent folder. Not only enforce this during project creation. Think about imported tenants via meshObject import.

Reply:
I've talked about this with Stefan. With this flag we only verify that a project is attached to a sub folder of the defined resource. If this is not the case then we call the update project function. After this step the project will be moved to the right place depend on the defined gcp function.